packages/cra-template-imodeljs/template/src/api/App.ts

```diff
@@ -0,0 +1,74 @@
+import { ClientRequestContext, Config } from "@bentley/bentleyjs-core";
+import { BrowserAuthorizationCallbackHandler, BrowserAuthorizationClient, BrowserAuthorizationClientConfiguration } from "@bentley/frontend-authorization-client";
+import { BentleyCloudRpcManager, BentleyCloudRpcParams } from "@bentley/imodeljs-common";
+import { FrontendRequestContext, IModelApp } from "@bentley/imodeljs-frontend";
+import { UrlDiscoveryClient } from "@bentley/itwin-client";
+import { UiComponents } from "@bentley/ui-components";
+import { IModelReadRpcInterface, IModelTileRpcInterface, RpcInterfaceDefinition } from "@bentley/imodeljs-common";
+
+/**
+ * Returns a list of RPCs supported by this application
+ */
+export function getSupportedRpcs(): RpcInterfaceDefinition[] {
+  return [
+    IModelReadRpcInterface,
+    IModelTileRpcInterface,
+  ];
+}
+
+// Boiler plate code
+export class iModeljsApp {
```

Comment:
Can you name this `App` instead of `iModeljsApp`?

Reply:
https://github.com/ashish-srivastava-dev/create-react-app/blob/cra-template-imodeljs/packages/cra-template-imodeljs/template/src/components/App.tsx 
This file also has an App class component and also imports iModeljsApp. So both App names would conflict.

Reply:
If we switch that to being a functional component, with hooks, instead of a class component then we wouldn't clash there. What do you think of making that change?

Reply:
I tried that change locally, but conflict still happens as both are React components with the same name and App component defined in api\App.ts is being consumed by App component defined in components\App.tsx.

Reply:
The iModeljsApp class has been renamed to Api as it contains static functions \ api and these are consumed in Components\App.tsx and index.tsx.